recipes/pycurl/python-pycurl/0001-disable-autodetection-of-libcurl-linkerflags.patch

```diff
@@ -0,0 +1,101 @@
+From 59f3305c3efce07fb3216cb5872614e8e5689aae Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Martin=20Hundeb=C3=B8ll?= <mnhu@prevas.dk>
+Date: Wed, 24 Feb 2016 15:12:00 +0100
+Subject: [PATCH] disable autodetection of libcurl linkerflags
+
+The call to curl-config produced "-L/lib" as linker flag, which is
```

Comment:
That sounds like a bug in curl-config, and should be fixed.  Is it using binconfig.oeclass properly?